code-for-canada/project-thundercat
# [Bug Fixes] Fixing Email Preview Gap and Keyboard Navigation

# Description

This changes in this PR are fixing the following bugs:
- Gap at the top of the email preview list
- Email preview keyboard navigation

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Screenshot

**Gap Bug**
---

**Before:**

![image](https://user-images.githubusercontent.com/23021242/55979997-779ef500-5c61-11e9-97c3-9bb47f24089d.png)

**After:**

![image](https://user-images.githubusercontent.com/23021242/55980052-9a310e00-5c61-11e9-86d7-8d41759382a6.png)

**Keyboard Navigation Bug**
---

**Mouse Over and Keyboard Focus:**

![image](https://user-images.githubusercontent.com/23021242/56231275-13f33e00-604c-11e9-93c3-5ed754bc264c.png)

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB Sample Test.
2.  Complete the _pre-test_ process.
3.  Open the _Inbox_ tab.
4.  Make sure the the gap bug is fixed.
5.  Also, make sure that you are able to navigate in the email preview list using the keyboard and everything look good (mouse over and focus).

# Checklist

Applicable for all code changes.
